cmd/ocitools/generate.go

```diff
@@ -58,6 +58,19 @@ var generateFlags = []cli.Flag{
 	cli.StringSliceFlag{Name: "seccomp-errno", Usage: "specifies syscalls to be added to list that returns an error"},
 	cli.StringFlag{Name: "template", Usage: "base template to use for creating the configuration"},
 	cli.StringSliceFlag{Name: "label", Usage: "add annotations to the configuration e.g. key=value"},
+	cli.Uint64Flag{Name: "cpu-shares", Usage: "CPU shares (relative weight (ratio) vs. other cgroups with cpu shares)"},
```

Comment:
On Tue, Aug 02, 2016 at 01:36:31PM -0700, hmeng-19 wrote:
> > @@ -58,6 +58,19 @@ var generateFlags = []cli.Flag{
> >  	cli.StringSliceFlag{Name: "seccomp-errno", Usage: "specifies syscalls to be added to list that returns an error"},
> >  	cli.StringFlag{Name: "template", Usage: "base template to use for creating the configuration"},
> >  	cli.StringSliceFlag{Name: "label", Usage: "add annotations to the configuration e.g. key=value"},
> > +	cli.Uint64Flag{Name: "cpu-shares", Usage: "CPU shares (relative weight (ratio) vs. other cgroups with cpu shares)"},
> 
> @wking , I tried to do that and eventually gave up. 
> Those descriptions are too long, and the output of `ocitools generate -h` will overwhelm.

I'm fine with abbreviating/excerpting here (and in the Go docs).  But
the wording should at least be related.